src/corehost/cli/redirected_error_writer.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Please rebase and update the license headers in the added files to match all the other source files.